src/translator/quality_estimator.h

```diff
@@ -0,0 +1,38 @@
+#ifndef SRC_BERGAMOT_QUALITY_ESTIMATOR_H_
+#define SRC_BERGAMOT_QUALITY_ESTIMATOR_H_
+#include <iostream>
+#include <vector>
+#include <map>
+#include <string>
+#include "annotation.h"
+#include "response.h"
+#include "intgemm/intgemm.h"
+#include "intgemm/aligned.h" 
+#include "intgemm/callbacks.h"
+
+namespace marian {
+namespace bergamot {
+
+class QualityEstimator {
+    private:
+        struct FeatureVector{
+            FeatureVector(float x, float y, float w, float z): 
+                feature_vector{x,y,z,w} {}
+            float feature_vector[4];
+        };
+        std::vector<float> stds, means, coefficients, intercept;
+        std::vector<std::map<std::tuple<size_t, size_t>, float>> mapping;
```

Comment:
Avoid using member variables as a means to pass temporary state between functions.

Reply:
I have removed this part in a further commit.